dev-tools/kubernetes/heartbeat/Dockerfile.debug

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.23.6 as builder
+FROM golang:1.24.0 as builder
```

Comment:
Go 1.24.1 is out, shall we use it instead of .0?

Reply:
let's get the `1.24.0` update merged in as this is currently blocking some fips related work and then focus on `1.24.1` update once the `golang-crossbuild` is fixed.